docs/sources/panels-visualizations/visualizations/datagrid/index.md

```diff
@@ -0,0 +1,88 @@
+
+### Editing data
+
+Data can be edited by double clicking on the cell that needs to be modified. This will activate the cell and allow you to edit the data. After editing the data, clicking anywhere outside the cell or pressing the `Enter` key will finalize the edit. To easily clear a cell of data, you can click on a cell to focus it and then press the `Delete` key.
+
+### Moving data
+
+Columns and rows can be moved around as desired. To move a column, click and hold the column header and drag it to the desired location. Releasing the mouse button will finalize the move. To move a row, click and hold the row selector from the number column situated on the left side and drag it to the desired location. Releasing the mouse button will finalize the move.
+
+### Selecting multiple cells
+
+You can select multiple cells by clicking on a single cell and dragging the mouse across others. This selection can be used to copy the data from the selected cells or to delete them using the `Delete` key.
+
+### Using the context menu
+
+To provide a more streamlined experience, the datagrid panel offers a context menu that can be accessed on right mouse click. Based on the state of the datagrid, the context menu will offer different options. The context menu can be accessed by right mouse clicking on a cell, column header, or row selector. Through the context menu, you can delete or clear rows and columns, remove all existing data (rendering the datagrid blank), or trigger the search functionality. Deleting a row or column will remove the data from the datagrid, while clearing a row or column will only remove the data from the cells, leaving the row or column intact.
+
+![Datagrid context menu](/media/docs/datagrid/screenshot-grafana-datagrid-context-menu.png)
+
+### Deleting/clearing multiple rows or columns
+
+To delete or clear multiple rows, you can select the desired rows through the checkboxes found on the left side number column and then right click to access the context menu. To select multiple consecutive rows, you must hold the `Shift` key pressed while clicking on the first and last row. To select certain rows, the `Ctrl` (or `Cmd`) key must be pressed while clicking on the desired rows. The same can be done for columns, by clicking on the column headers and holding the `Shift` or `Ctrl` (`Cmd`) keys and then opening the context menu to apply the desired action.
+
+To delete all rows, use the `Select all` checkbox found on the top left corner of the datagrid. This will select all rows and allow the user to delete them through the context menu.
+
+### Searching data
+
+The search functionality can be triggered from the context menu and will allow you to find keywords within the dataset. The searchbox can then be closed either by clicking the `x` button or pressing the `Escape` key.
+
```

Comment:
The second sentence seems like overkill to me, which then renders this a one-sentence section, which isn't ideal. I'd kill this section and I've made the first sentence part of the search bullet point in the context menu section.
```suggestion
```